### PR TITLE
Align logpush cURL examples to be consistent

### PR DESCRIPTION
**Pull request by cmackenzie1**
Aligning the cURL examples to use the same fields and user provided variable notation.

### CODE REVIEW

File: products/logs/src/content/reference/logpush-api-configuration/examples/example-logpush-curl/index.md [hunk 1]
```diff
@@ -340,14 +340,13 @@ curl -s -X GET 'https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logpush/job
 }
 ```
 
-Next, edit the `logpull_options` as desired and create a `PUT` request. The following example enables the `CVE-2021-44228` redaction CVE-2021-44228 option.
+Next, edit the `logpull_options` as desired and create a `PUT` request. The following example enables the `CVE-2021-44228` redaction option.
 
 ```bash
-curl -s -X PUT 'https://api.cloudflare.com/client/v4/zones/${ZONE_TAG}/logpush/jobs/${JOB_ID}/' \
+curl -s -X PUT 'https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logpush/jobs/<JOB_ID>/' \
```
Comment (angelampcosta) [inline products/logs/src/content/reference/logpush-api-configuration/examples/example-logpush-curl/index.md]: According to support: "We have seen customers experiencing 404 errors when they run the above API call. We were able to identify that there is a slash(/) at the end of JOB_ID which is causing issues. The customer's issue was resolved when they run the API call by removing the Slash". Probably the slash after JOB_ID needs to be removed.

File: products/logs/src/content/reference/logpush-api-configuration/examples/example-logpush-curl/index.md [hunk 2]
```diff
@@ -294,12 +294,12 @@ curl -s -X GET https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logpush/jobs
...
       "dataset": "http_requests",
       "enabled": false,
       "name": "<DOMAIN_NAME>",
-      "logpull_options": "fields=ClientIP,ClientRequestHost,ClientRequestMethod,ClientRequestURI,EdgeEndTimestamp,EdgeResponseBytes,EdgeResponseStatus,EdgeStartTimestamp,RayID&timestamps=rfc3339&sample=0.06",
-      "destination_conf": "s3://<BUCKET_PATH_HTTP_REQUESTS>?region=us-west-2",
+      "logpull_options": "fields=ClientIP,ClientRequestHost,ClientRequestMethod,ClientRequestURI,EdgeEndTimestamp,EdgeResponseBytes,EdgeResponseStatus,EdgeStartTimestamp,RayID&timestamps=rfc3339",
+      "destination_conf": "s3://<BUCKET_PATH>?region=us-west-2",
       "last_complete": "2019-09-24T21:15:00Z",
       "last_error": null,
       "error_message": null
```
Comment (angelampcosta) [inline products/logs/src/content/reference/logpush-api-configuration/examples/example-logpush-curl/index.md:305]: I think there shouldn't be a comma on line 306 and line 307 should have another }  to close the curly brackets opened on line 281.

File: products/logs/src/content/reference/logpush-api-configuration/examples/example-logpush-curl/index.md [hunk 3]
```diff
@@ -258,7 +258,7 @@ curl -s -X GET https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logpush/jobs
      "dataset": "http_requests",
      "enabled": true,
      "name": "<DOMAIN_NAME>",
-     "logpull_options": "fields=RayID,EdgeStartTimestamp&timestamps=rfc3339",
+     "logpull_options": "fields=ClientIP,ClientRequestHost,ClientRequestMethod,ClientRequestURI,EdgeEndTimestamp,EdgeResponseBytes,EdgeResponseStatus,EdgeStartTimestamp,RayID&timestamps=rfc3339",
      "destination_conf": "s3://<BUCKET_PATH>?region=us-west-2",
      "last_complete": null,
      "last_error": null,
```
Comment (angelampcosta) [inline products/logs/src/content/reference/logpush-api-configuration/examples/example-logpush-curl/index.md:264]: The { on line 281 doesn't close.

File: products/logs/src/content/reference/logpush-api-configuration/examples/example-logpush-curl/index.md (hunk 1 above)
Comment (tanushree-sharma) [reply products/logs/src/content/reference/logpush-api-configuration/examples/example-logpush-curl/index.md]: Confirmed that I'm seeing the same 404 error with the / at the end of the line. @cmackenzie1 do you want to make that change in this PR?

Comment (cmackenzie1) [reply products/logs/src/content/reference/logpush-api-configuration/examples/example-logpush-curl/index.md]: Updated it!

**Review by soheiokamoto — APPROVED** [review]

**Pull request merged**